### ISSUE TITLE
ModOrganizer2 Crash with Windows 10 Pro Game Skyrim

### ISSUE THREAD

**Issue by inire** [TanninOne/modorganizer#529]
Hello,

I recently installed a fresh Windows 10 Professional on a new system, and have been trying to get ModOrganizer2 running. I am using ModOrganizer2 alpha 3 (2.0.3a) when launching after choosing the Skyrim option and the Non-portable install.

I installed both Steam and ModOrganizer2 in the c:\Games directory, which my logged in user has full rights to.

Attached please find the Crash Dump, output from Speccy for my system, and the output from Belarc Advisor showing my installed software. Note that I have both Skyrim and Fallout 4 installed on this system, but I was trying to set up Skyrim again, as I just recently reinstalled Windows 10.

Perhaps I am missing a required library of some kind as I am using a new install of Windows 10?

I tried running the application both as Administrator and as regular user.

Note also that my system has IPv6 and IPv4 running, and the older version of ModOrganizer (1.3.11) sees the Ethernet adapter with the IPv6 address usually when starting.

Also note that I have OpenSSL 32/64 bit installed on my system for some other uses with GOW and curl/wget. When I launched the 1.3.11 version originally it failed to launch with an error about DH entry points in the library ssleay32.dll. I fixed this by upgrading to 1.0.2f version of OpenSSL. This upgraded DLL did NOT fix the ModOrganizer 2.0.3a crash however.

[ModOrgainzer2Dump1-shartmann1_30_2016-1.zip](https://github.com/TanninOne/modorganizer/files/110860/ModOrgainzer2Dump1-shartmann1_30_2016-1.zip)

**Comment by Sir1Twist** [comment]
Mod Organizer crashes when you try to do almost anything, correct? There is a file you need to move outside of the hierarchy of Mod Organizer. It is bsa_extractor.dll. Moving that file should solve your problems.

**Comment by inire** [comment]
Hello,

I tested this by renaming the file located in the plugins directory from bsa_extractor.dll to bsa_extractor.old and ModOrganizer2 still crashed.

I then moved the file completely out of the directory tree to the desktop, and the crash still happened.

Thanks for the advice. Any other possibilities? Perhaps the crash dump from moving the file out is different now, I have attached the updated crash dump in case moving the bsa_extractor.dll file out moved my failure one step further down the crash tree.

Thanks.
[ModOrganizer2dump2-shartmann1_30_2016-2.zip](https://github.com/TanninOne/modorganizer/files/111095/ModOrganizer2dump2-shartmann1_30_2016-2.zip)
